warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_schedule_validation_errors.sql

```diff
@@ -0,0 +1,35 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+validation_notices AS (
+    SELECT * FROM {{ ref('fct_daily_schedule_feed_validation_notices') }}
+),
+
+validation_errors_by_day AS (
+    SELECT
+        feed_key,
+        date,
+        SUM(total_notices) AS sum_total_notices
+    FROM validation_notices
+    WHERE severity = "ERROR"
+    GROUP BY 1, 2
+),
+
+int_gtfs_quality__no_schedule_validation_errors AS (
+    SELECT
+        idx.date,
+        idx.feed_key,
+        {{ no_validation_errors() }} AS check,
+        {{ compliance() }} AS feature,
+        CASE
+            WHEN sum_total_notices > 0 THEN "FAIL"
+            WHEN sum_total_notices = 0 THEN "PASS"
```

Comment:
I think it's really about 99.9% and I imagine it's primarily cases of invalid zips, etc. I do think a `null` is appropriate because we can't always know exactly why we are missing output for the record. I've gone back and forth about sentinel values and over time I've come to dislike them, though I will defer to @owades on the specifics here.

Reply:
My opinion is it depends on what the most common reasons for validator failures would be. If a common issue is invalid zips, then we could leave the result here as null, and catch the invalid zips in a different check. 

Although, my preference would be for the validator to run and catch the invalid zip as an error.